packages/core/src/render3/util/defer.ts

```diff
@@ -115,6 +116,15 @@ function findDeferBlocks(node: Node, lView: LView, results: DeferBlockData[]) {
         renderedLView[TVIEW].firstChild,
         rootNodes,
       );
+    } else if (hydrationState === 'dehydrated') {
```

Comment:
We should probably inline the call, so that we don't do extra computation:
```suggestion
    } else if (inferHydrationState(tDetails, lDetails, registry) === 'dehydrated') {
```

Reply:
I intended to `hydrationState` further down the code where `inferHydrationState` was also used.